test/harry/main/org/apache/cassandra/harry/model/BytesPartitionState.java

```diff
@@ -62,31 +65,36 @@ private BytesPartitionState(Factory factory, Clustering<ByteBuffer> key)
         this.state = factory.partitionState(key);
     }
 
-    public void deleteRow(Clustering<ByteBuffer> clustering)
+    public void deleteRow(Clustering<ByteBuffer> clustering, long ts)
     {
         long cd = factory.clusteringCache.deflateOrUndefined(clustering);
         if (MagicConstants.UNSET_DESCR == cd)
             return;
-        state.delete(cd, MagicConstants.NO_TIMESTAMP);
+        deleteRow(cd, ts);
     }
 
-    public void deleteColumns(Clustering<ByteBuffer> clustering, Set<Symbol> columns)
+    private void deleteRow(long cd, long ts)
+    {
+        state.delete(cd, ts);
+    }
```

Comment:
not needed in this patch but is to make way for multi cell reconcile that `BATCH` needs...